src/styles/globals.css

```diff
@@ -93,4 +93,8 @@
     .btn.toRepertoire {
         @apply rounded bg-green-500 px-12 py-2 font-bold text-white hover:bg-green-300 disabled:opacity-25 disabled:bg-gray-500 disabled:text-black
     }
+
+    .btn.toSuggestions {
+        @apply rounded bg-red-400 px-12 py-2 font-bold text-white hover:bg-red-300 disabled:opacity-25 disabled:bg-gray-500 disabled:text-black
+    }
```

Comment:
These are only used in one place right? Is it necessary to create separate classes for them?
Also, most of the classes except for the bg & bg on hover are identical. If you want to keep them, I'd suggest creating a 'parent' class for them with the following classes: `@apply rounded px-12 py-2 font-bold text-white disabled:opacity-25 disabled:bg-gray-500 disabled:text-black` and then just add the colors in the code where they are used